metavalidator/all_test.go

```diff
@@ -0,0 +1,69 @@
+package metavalidator
```

Comment:
It looks like a few of the other packages were recently switched to the `_test` testing package convention so they can verify things via only exported functionality. Not sure if this should be updated as well.